docs/modules/ROOT/pages/project-creator.adoc

```diff
@@ -0,0 +1,110 @@
+
+* If you want to force it to fetch the latest updates from repositories while listing them you need to add `--refresh` flag.
+
+* Keep in mind that `--local` and `--refresh` flags are mutually exclusive which means they cannot be used together in a single `template list` command.
+
+Let's create our first project by using link:https://github.com/hazelcast-templates/go-client-getting-started[go-client-getting-started] template. It includes two following files that has special extensions.
+
+```
+go-client-getting-started
+├── README.md.keep
+├── config.go.template
+└── ...
+```
+
+`.keep` and `.template` extensions are special ones for project creator. `.template` extension specifies that it is a link:https://pkg.go.dev/text/template[Go template] file, and it includes placeholders and actions. `.keep` extension is used to keep the files that are required for the created project. For example, the `README.md` file in the template will be replaced with `README.md.keep` one when the project is created.
+
+Let's look a bit deeper to `config.go.template` to see what placeholders are included:
+
+* config
+* cluster_name
+* cluster_discovery_token
+* cluster_address
+* ssl_key_path
+* ssl_cert_path
+* ssl_key_password
+* cluster_api_base
+
+are the ones that are used in this template. More placeholders are listed in link:clc-project.adoc[project creator reference documentation].
+
+To create a project, run the following command:
+
+[source,bash]
+----
+clc -c <your-config> project create go-client-getting-started
+----
+
+The project will be created in the current working directory. If you want it to be created in a different directory, you can use `--output-dir` flag. After creating the project, you can directly run it without any effort, because placeholders already configured the project based on the given configuration.
+
+== Creating and Using Your Own Templates
+
+It is also possible users to create their own templates and use them. While creating your own template the following rules should be kept in mind:
+
+* Templates are in link:https://pkg.go.dev/text/template[Go template] format.
+* You can create a "template.yaml" file in template's root directory and add it a "defaults" node to add default values.
+* Template files must have the ".template" extension.
+* Files with "." and "_" prefixes are ignored unless they have the ".keep" extension.
+* All files with ".keep" extension are copied by stripping the ".keep" extension.
+* Other files are copied verbatim.
+*Properties are read from the following resources in order:
++
+1. defaults node of `templates.yaml` (keys cannot contain punctuation)
+2. `config.yaml`
+3. User passed key-values in the "KEY=VALUE" format. The keys can only contain lowercase letters, numbers and the underscore character.
```

Comment:
User-passed key-values in `KEY=VALUE` format. 
+
NOTE: The keys must contain only lowercase, numerical, and underscore characters.

Reply:
https://github.com/hazelcast/clc-docs/pull/7/commits/08b9655e9467f9fae27b513e7c263e2880adcf9f

Reply:
You need a plus (+) character on the empty line between the list item and the NOTE

Start the numbered lists with two periods (..) rather than 1. etc; the double dot will nest it properly

Reply:
https://github.com/hazelcast/clc-docs/pull/7/commits/d1ee9048567f844b16f959fc35b68c6db2fc6b85